main/key_handler.go

```diff
@@ -150,7 +151,7 @@ func registerPublicKey(p *ExtendedProtocol, uid uuid.UUID, pubKey []byte, keySer
 
 // submitCSR submits a X.509 Certificate Signing Request for the public key to the identity service
 func submitCSR(p *ExtendedProtocol, uid uuid.UUID, subjectCountry string, subjectOrganization string, identityService string) error {
-	log.Printf("%s: submitting CSR to identity service: %s", uid, identityService)
+	log.Debugf("%s: submitting CSR to identity service", uid)
```

Comment:
I always liked using these messages to see if all clients were recognized, but I understand why they should not stay.
I'm happy about the Infof() in line 34, that has enough information to debug common problems for me.
Will this information still be available on Default debugging level (in production)?

Reply:
i lowered the logging level to "debug" since it was a bit much when testing with 2000 devices ;) since we have the "debug" flag in the configuration, you will still see the message though